src/test/com/aca/homework/week9/generic/list/ArrayListTestForInteger.java

```diff
@@ -0,0 +1,33 @@
+package com.aca.homework.week9.generic.list;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ArrayListTestForInteger {
```

Comment:
please create a single ArrayListTest class and instantiate the test subject in every method